pkg/controller/backend.go

```diff
@@ -168,6 +168,7 @@ func (agent *Agent) agentWorker() {
 		}
 		decl := agent.createTenantAS3Declaration(rsConfig)
 
+		log.Debugf("\n\n\n[AS3] [DECL]: \t%v\n\n\n\n", decl)
```

Comment:
Can we remove this log as it will print complete declaration in logs, it may cause security threat?